python/tank/pipelineconfig.py

```diff
@@ -310,6 +335,7 @@ def convert_to_site_config(self):
         Converts the pipeline configuration into the site configuration.
         """
         self._update_pipeline_config({"project_id": None})
+        self._project_id = None
```

Comment:
What is going on here? Doesn't update_pipeline_config already set project_id to None?